for l in range(9,10):
    pass             # nothing happen it stops code to get error 